### DISCUSSION TITLE
配置了样式前缀，配置了babel-loader样式不生效

### DISCUSSION THREAD

**Discussion by daisymark** [alibaba/formily#2730] [Q&A 2.x] [answered]
![image](https://user-images.githubusercontent.com/6830835/148195040-80f3c784-a266-4192-be3f-cbadebabc68d.png)

![image](https://user-images.githubusercontent.com/6830835/148195142-318954cf-c20f-40f3-ae8f-2810cc4ec3ef.png)

![image](https://user-images.githubusercontent.com/6830835/148195163-2ac42a07-a068-446b-9b72-305fc354c677.png)

![image](https://user-images.githubusercontent.com/6830835/148195176-2a14216a-81a6-49f6-b32a-30ce381f9069.png)

**Comment by janryWang** [comment, marked as answer]
看官网按需打包方案

**Reply by daisymark** [reply]
![image](https://user-images.githubusercontent.com/6830835/148197415-7a385eb3-bf33-47b8-a543-3a91e2d98635.png)
配置了

**Reply by daisymark** [reply]
![image](https://user-images.githubusercontent.com/6830835/148198678-af72176d-899e-4ee2-8468-e11ff683e4f0.png)
也执行了

**Reply by daisymark** [reply]
![image](https://user-images.githubusercontent.com/6830835/148199146-5244f4a0-372f-4470-b1d7-aa1cb0a7391f.png)
